webapp/components/NotificationsTable/NotificationsTable.vue

```diff
@@ -61,41 +96,38 @@ export default {
     notifications: { type: Array, default: () => [] },
   },
   computed: {
+    ...mapGetters({
+      currentUser: 'auth/user',
+    }),
     fields() {
       return {
         icon: {
           label: ' ',
-          width: '5',
+          width: '3%',
         },
         user: {
-          label: this.$t('notifications.user'),
-          width: '45%',
+          label: ' ',
+          width: '25%',
         },
         post: {
-          label: this.$t('notifications.post'),
+          label: ' ',
           width: '25%',
         },
         content: {
-          label: this.$t('notifications.content'),
-          width: '25%',
+          label: ' ',
+          width: '47%',
         },
       }
     },
+    notificationsData() {
+      const data = []
+      this.notifications.forEach(notification => {
+        data.push(extractNotificationDataOfCurrentUser(notification, this.currentUser))
+      })
```

Comment:
```suggestion
      return this.notifications.map(notification => {
        extractNotificationDataOfCurrentUser(notification, this.currentUser)
      })
 ```
```

Reply:
I think this should work, oder?

Reply:
Hell, yes. That's one of the exam questions for my students: How to eliminate unnecessary temporary states like `data` in this case..

Reply:
You are really close to it. @mattwr18 
Just leave the curly braces away or put a `return` in front of `extractNotificationDataOfCurrentUser `.

@roschaefer Now I do really understand `eliminate unnecessary temporary states`!